raster-test/src/test/scala/geotrellis/raster/io/geotiff/GeoTiffMultiBandTileSpec.scala

```diff
@@ -110,6 +110,50 @@ class GeoTiffMultiBandTileSpec extends FunSpec
     }
   }
 
+  describe("MutliBand subset methods") {
+
+    it("subset should be inexpensive") {
+      val tile0 = MultiBandGeoTiff(geoTiffPath("3bands/int32/3bands-striped-pixel.tif"))
+      val tile1 = tile0.subset(List(1, 2, 0))
+
+      tile0.band(0) should be (tile1.band(2))
+      tile0.band(1) should be (tile1.band(0))
+      tile0.band(2) should be (tile1.band(1))
```

Comment:
Seems like it's just doing the object equals anyway, but it would be better to explicitly state in the test that it's an object reference equality, +1